azure-webapp-maven-plugin/src/main/java/com/microsoft/azure/maven/webapp/handlers/artifact/JarArtifactHandlerImpl.java

```diff
@@ -77,21 +77,15 @@ public void publish(DeployTarget deployTarget) throws IOException, MojoExecution
         final File jar = getJarFile();
         assureJarFileExisted(jar);
 
-        prepareDeploymentFiles(deployTarget, jar);
+        prepareDeploymentFiles(jar);
 
         super.publish(deployTarget);
     }
 
-    protected void prepareDeploymentFiles(DeployTarget deployTarget, File jar) throws IOException {
+    protected void prepareDeploymentFiles(File jar) throws IOException {
         final File parent = new File(stagingDirectoryPath);
         parent.mkdirs();
-
-        if (StringUtils.isNotEmpty(linuxRuntime)) {
-            Files.copy(jar, new File(parent, DEFAULT_LINUX_JAR_NAME));
-        } else {
-            Files.copy(jar, new File(parent, jar.getName()));
-            WebAppUtils.generateWebConfigFile(deployTarget, jar.getName(), stagingDirectoryPath, log);
-        }
+        Files.copy(jar, new File(parent, DEFAULT_LINUX_JAR_NAME));
```

Comment:
DEFAULT_LINUX_JAR_NAME should be renamed to DEFAULT_JAR_NAME ?

Reply:
DEFAULT_LINUX_JAR_NAME and DEFAULT_JAVA_SE_JAR_NAME are same?

Reply:
yes, we need rename the executable jar file to `app.jar`, both for windows and linux

Reply:
Renamed to `DEFAULT_APP_SERVICE_JAR_NAME`